test/builder/eventlistener.go

```diff
@@ -204,3 +204,16 @@ func EventListenerTriggerValidateParam(name, value string) EventListenerTriggerV
 		)
 	}
 }
+
+// EventListenerTriggerValidateParam adds a param name to the TriggerValidate.
+func EventListenerTriggerInterceptor(name, version, kind string) EventListenerTriggerOp {
```

Comment:
I don't know if the intent is to restrict interceptors to the same namespace since there is no parameter for it?

Reply:
No, you can specify a namespace, but it defaults to the namespace of the EL. 
Added a namespace param to the builder.